File: Code/Practice.c
#include <stdio.h>
int main()
{
    int a[100], i, j, n, temp;
    printf("Enter size of array: ");
    scanf("%d", &n);

    printf("Enter elements of array: ");
    for (i = 0; i < n; i++)
    {
        scanf("%d", &a[i]);
    }

    for (i = 0; i < n - 1; i++)
    {

        int smallest = i;
        for (j = i + 1; j < n; j++)
        {
            if (a[j] < a[smallest])
            {
                smallest = j;
            }
        }
        temp = a[i];
        a[i] = a[smallest];
        a[smallest] = temp;
    }
    printf("Elements in Ascending order is\t");
    for (i = 0; i < n; i++)
    {
        printf("%d\t", a[i]);
    }
}

File: Code/Practice.js
function bubbleSort(arr) {
    let n = arr.length;
    let x;

    for (let i = 0; i < n; i++) {
        for (let j = i + 1; j < n; j++) {
            if (arr[i] > arr[j]) {
                x = arr[i];
                arr[i] = arr[j];
                arr[j] = x;
            }
        }
    }

    console.log("The ascending order of the elements is:");
    for (let i = 0; i < n; i++) {
        console.log(arr[i]);
    }
}

function main() {
    let arr = [];
    let n;

    n = parseInt(prompt("Enter the number of elements:"));

    console.log("Enter elements:");
    for (let i = 0; i < n; i++) {
        arr.push(parseInt(prompt(`Enter element ${i + 1}:`)));
    }

    bubbleSort(arr);
}

main();
